docs/topics_5/proc_web-downloading-logs-cli.adoc

```diff
@@ -0,0 +1,42 @@
+// Module included in the following assemblies:
+//
+// * docs/web-console-guide_5/master.adoc
+
+[id="proc_web-downloading-logs-cli_{context}"]
+= Downloading pod logs using the CLI
```

Comment:
Verbose. Change to "Downloading logs by using the CLI"

We use "by using" instead of "using" or "with" to make localization easier.